includes/UserAdd.php

```diff
@@ -147,11 +154,18 @@
 
 	if( ! $stmt->execute() )
 	{
-		//echo "Error : $sqlConn->errno : $sqlConn->error <br>";
-		echo "Error creating database for user, contact Admin";
+		
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->